src/htm/regions/TestNode.cpp

```diff
@@ -216,6 +216,7 @@ void TestNode::compute() {
 Spec *TestNode::createSpec() {
   auto ns = new Spec;
 
+  ns->name = "TestNode";
   ns->description = "TestNode. Used as a plain simple plugin Region for unit tests only. "
```

Comment:
nit, do we need the TestNode , or could we simply use any of the SP,TMRegions?

Reply:
Many of the unit tests use this as a "generic" region.  It contains a little of everything.  Not all regions actually use all features.